src/js/Controllers/DisplayCapabilities.js

```diff
@@ -137,6 +137,13 @@ let capabilities = {
 						"resolutionWidth": 360,
 						"resolutionHeight": 360
 					}
+				},
+				{
+					"name": "alertIcon",
```

Comment:
Oddly, `imageTypeSupported` is mandatory in the Mobile API, but not the HMI API. Please include this parameter

Reply:
included in https://github.com/smartdevicelink/generic_hmi/pull/177/commits/9e4b4aea1dc8955cee50bf75c6e6b3d2d14ded7d